.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 references:
-  - &CERT_KEY certs-cache-{{ checksum "tools/ssl/Makefile" }}-{{ checksum "cert_date" }}-v5
+  - &CERT_KEY certs-cache-{{ checksum "certs_cache_key" }}
```

Comment:
No no, here I'd still keep the Makefile checksum, if we ever change the makefile we'll probably want new keys, I would just not hash the yyyy-weekXX part, so it's readable 🙂

Reply:
Ah, wait, that's done on the bash script, I commented without reading to the end 😅